docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -179,3 +179,7 @@ Property Name               Default Value    Description
 ``bloomfilter``             ``NONE``         Bloomfilter to use. Valid values are ``NONE`` (default), ``ROW``, or ``ROWCOL``.
 =========================== ================ ==============================================================================================================
 
+Limitations
```

Comment:
Not at the moment, because the "SQL Support" section should be a more complete discussion of what SQL statements are & aren't allowed. We still have to build that out for this connector (and several others) so "Limitations" makes more sense for now.